pkgs/by-name/ge/gema/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchgit,
+  installShellFiles,
```

A preference seems to exist to format the package attributes with leading comma: 

```suggestion
{ lib
, stdenv
, fetchgit
, installShellFiles
```

---

Thanks! Done. I foolishly followed what nixfmt-rfc-style did.